Dockerfile

```diff
@@ -1 +1,14 @@
-FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/hello-world:latest
+FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base:20210701-2f4a4c7
+
+RUN  yum -y install java-1.8.0-openjdk
+
+ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk-1.8.0.292.b10-1.el7_9.x86_64
```

Comment:
same comment here as on the other one; use the symlink `/usr/lib/jvm/java-1.8.0-openjdk`.

Reply:
JAVA_HOME environment variable is required for tomcat9 and Solr. So dockerfile for tomcat9 and solr inherits this build image of java8 does not require to explicitly mention env variable.

Reply:
Don't we need an equal sign after JAVA_HOME?  @naveen-armedia Dave is asking you to use the symlink instead of the exact path (for future flexibility).

Reply:
`ENV JAVA_HOME /usr/lib/jvm/java-1.8.0-openjdk`

Reply:
updated with systemlink